app/web/features/profile/view/Overview.tsx

```diff
@@ -101,11 +105,26 @@ function DefaultActions({
       <MessageUserButton user={user} setMutationError={setMutationError} />
       <FriendActions user={user} setMutationError={setMutationError} />
 
-      <FlagButton
-        className={classes.flagButton}
-        contentRef={`profile/${user.userId}`}
-        authorUser={user.userId}
-      />
+      <div className={classes.modButtons}>
+        <FlagButton
+          contentRef={`profile/${user.userId}`}
+          authorUser={user.userId}
+        />
+
+        {accountInfo?.isSuperuser && (
+          <IconButton
+            aria-label={t("profile:view_in_admin_console")}
+            onClick={() =>
+              router.push(
+                `${process.env.NEXT_PUBLIC_CONSOLE_BASE_URL}/admin/user/${user.username}`
+              )
+            }
+            color="primary"
+          >
+            <SettingsIcon />
+          </IconButton>
+        )}
+      </div>
```

Comment:
We should probably add a frontend test for this since it has to do with permissions and that's more important than the average UI feature.